core/src/main/java/com/mapzen/android/graphics/OverlayManager.java

```diff
@@ -814,4 +871,13 @@ private MapData addPointToMarkerMapData(Marker marker) {
     findMe.setActivated(false);
     return false;
   }
+
+  @Override public boolean onRotate(float x, float y, float rotation) {
+    final View compass = mapView.findViewById(R.id.mz_compass);
```

Comment:
Ok, I tried to replace that in onFling() and onPan() with mapView.getFindMe(), but two tests onFling_shouldDeactivateFindMe() and onPan_shouldDeactivateFindMe() are failing because there is a null pointer exception on findeMe.setActivated(false);. I don't know how mocking works but this seems to be a problem that MapView stored in OverlayManager has field findMe not initialized.